EXE/subtitles.xml

```diff
@@ -187,43 +190,43 @@
+<Static AudioFile="mod\m04\m04_final_0010_alaric.wav" TextArea="Cinematic">Awesome, Trent! Who knew you'd be so good at finding clients and closing deals?</Static>
+<Static AudioFile="mod\m04\m04_final_0020_trent.wav" TextArea="Cinematic">I'm also good at making enemies and getting close to being murdered.</Static>
+<Static AudioFile="mod\m04\m04_final_0030_alaric.wav" TextArea="Cinematic">Oh come on... You just started your freelancing career, a complete newbie, and now you're taking jobs for the government with appropriate payments!</Static>
+<Static AudioFile="mod\m04\m04_final_0040_trent.wav" TextArea="Cinematic">Mhm, one of those governments have already put a price on my head. And oh do I hope they forget about me soon with everything that's going on there.</Static>
+<Static AudioFile="mod\m04\m04_final_0050_alaric.wav" TextArea="Cinematic">Seriously, the Rheinland military will most likely fall apart soon. They won't bother looking for us.</Static>
 <Static AudioFile="mod\m04\m04_final_0060_trent.wav" TextArea="Cinematic">You seem very confident. Come on, Alaric, what are you hiding from me?</Static>
 <Static AudioFile="mod\m04\m04_final_0070_alaric.wav" TextArea="Cinematic">That Miss Hatcher, do you know who she is?</Static>
-<Static AudioFile="mod\m04\m04_final_0080_trent.wav" TextArea="Cinematic">Well, of course I do! I’ve known her for a whole three hours! Screw you, Alaric.</Static>
-<Static AudioFile="mod\m04\m04_final_0090_alaric.wav" TextArea="Cinematic">She’s an officer in one of Liberty’s secret service agencies.</Static>
+<Static AudioFile="mod\m04\m04_final_0080_trent.wav" TextArea="Cinematic">Well, of course I do! I've known her for a whole three hours! Screw you, Alaric.</Static>
+<Static AudioFile="mod\m04\m04_final_0090_alaric.wav" TextArea="Cinematic">She's an officer in one of Liberty's secret service agencies.</Static>
 <Static AudioFile="mod\m04\m04_final_0100_trent.wav" TextArea="Cinematic">God dammit! Not again...</Static>
-<Static AudioFile="mod\m04\m04_final_0110_alaric.wav" TextArea="Cinematic">You don’t get it. It’s such a special and secret service, the other ones don’t even suspect that it exists.</Static>
+<Static AudioFile="mod\m04\m04_final_0110_alaric.wav" TextArea="Cinematic">"You don't get it. It's such a special and secret service, that not even their own counterparts in their own government know they exist."</Static>
 <Static AudioFile="mod\m04\m04_final_0120_trent.wav" TextArea="Cinematic">Then how do you know that, Alaric?</Static>
-<Static AudioFile="mod\m04\m04_final_0130_alaric.wav" TextArea="Cinematic">А... E-e-e...</Static>
-<Static AudioFile="mod\m04\m04_final_0140_trent.wav" TextArea="Cinematic">I can't believe I got into this botch again!</Static>
+<Static AudioFile="mod\m04\m04_final_0130_alaric.wav" TextArea="Cinematic">Ah... E-e-e...</Static>
+<Static AudioFile="mod\m04\m04_final_0140_trent.wav" TextArea="Cinematic">I can't believe I got into this mess again!</Static>
```

Comment:
Is botch wrong here or is it just too fancy?

Reply:
Depends what nationality Trent leans in this incarnation, "Botch" may be more common in England, but in this fairly "American-English" Trent Stereotype, (I think?) it sounds like something that no Trent would say in English, to me, in this type of line.
I can change it based on any reason by anyone else's preference.
Give me more feedback.
"I can't believe I got into this mess again!" sounds non-cussy, yet impactful enough to me.
I agree that the line could potentially be improved or restructured, or perhaps "Shit," instead of "Mess"?

Reply:
The original Trent was rather of Bretonian origin. Thus my original choice of words. However, it's not the same Trent, so we can take any direction we'd like. "Shit" does sound more appealing, as the game's fanbase are mostly mature. Jeidar does add cussing in the Russian version, so we might just go with that.

Reply:
I changed the PR back to original "botch."
It may change in future, but now that you explained a bit of Lore with this "Original Trent" concept it seems fitting.  I had been reading through the script, thinking only of "Trent's Voice" in CD-ROM Freelancer 2003.
Thank you.